docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/cinema_service
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+
+  db:
+    image: postgres:16.0-alpine3.17
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
```

Comment:
Ensure that the `PGDATA` environment variable is correctly set in your environment to avoid issues with data persistence.